app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -124,6 +124,7 @@ def destroy
       end
 
       def link
+        @link = true
```

Comment:
You don't need to set `@link` here, as you can use `action_name` and `controller_name` in the view to distinguish between different controllers and actions.